### PR TITLE
New December Releases

### PR DESCRIPTION
**Pull request by holgerd77**
Releases on the vast majority of monorepo libraries - see labels for full list - with some awaited bug fixes, optimizations, also including EIP-4895 Withdrawals support.

Roughly targeted for a release on Thursday (Dec 8) or Friday, eventually also Monday or Tuesday after the weekend.

Anyone feeling the need for anything additionally to be included please drop a note (here or on Discord).

### CODE REVIEW

File: packages/client/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,32 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+This releases includes various bug fixes and optimizations discovered and applied along the run of the [Shandong](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2316) Pre-Shanghai testnet (https://github.com/ethereumjs/ethereumjs-monorepo/pull/2316) (now being deprecated) as well as new RPC methods implemented along the way.
+
+Furthermore this client release can now run chain including blocks with `EIP-4895` beacon chain withdrawals respectively execute on EIP-4895 activating hardforks, see PRs [#2353](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2353) and [#2401](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2401).
+
+### New RPC Methods
+
+- `eth_getBlockTransactionCountByNumber`, PR [#2379](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2379)
+- `eth_gasPrice`, PR [#2396](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2396)
+- `eth_feeHistory`, PR [#2413](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2413)
```
Comment (acolytec3) [inline packages/client/CHANGELOG.md:19]: This PR was never merged so we should remove this.


We should also consider merging [#2444](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2444) and noting it here.

Comment (acolytec3) [reply packages/client/CHANGELOG.md:19]: #2444 has been merged so we should add that here.  Something like:
```
- `debug_traceTransaction`, PR [#2444](https://github.com/ethereumjs/ethereumjs-monorepo/pull/2444)
```

**Review by acolytec3 — APPROVED** [review]
Looks great!  Excited to get this one out.

**Review by g11tech — APPROVED** [review]
lgtm!

**Comment by holgerd77** [comment]
Just published the following releases on npm:
- `@ethereumjs/util` [v8.0.3](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Futil%408.0.3)
- `@ethereumjs/common` [v3.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fcommon%403.0.2)
- `@ethereumjs/trie` [v5.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftrie%405.0.2)
- `@ethereumjs/tx` [v4.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Ftx%404.0.2)
- `@ethereumjs/block` [v4.1.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblock%404.1.0)
- `@ethereumjs/statemanager` [v1.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fstatemanager%401.0.2)
- `@ethereumjs/devp2p` [v5.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fdevp2p%405.0.2)
- `@ethereumjs/ethash` [v2.0.2](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fethash%402.0.2)
- `@ethereumjs/blockchain` [v6.1.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fblockchain%406.1.0)
- `@ethereumjs/evm` [v1.2.3](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fevm%401.2.3)
- `@ethereumjs/vm` [v6.3.0](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fvm%406.3.0)
- `@ethereumjs/client` [v0.6.6](https://github.com/ethereumjs/ethereumjs-monorepo/releases/tag/%40ethereumjs%2Fclient%400.6.6)

**Pull request merged**